node/src/actors/chain_manager/handlers.rs

```diff
@@ -238,7 +239,10 @@ impl Handler<AddCandidates> for ChainManager {
     fn handle(&mut self, msg: AddCandidates, _ctx: &mut Context<Self>) {
         // AddCandidates is needed in all states
         for block in msg.blocks {
-            self.process_candidate(block)
+            let block_hash = block.hash();
+            self.process_candidate(block);
+            info!("Block {} candidate proposed",
```

Comment:
We are trying to avoid using log functions without the crate prefix because it's easier to locate them with a simple grep: `log::`. So instead of using just `info!("...")` it would be better to use `log::info!("....")`.

Reply:
Hmm, I see. I will change that right away.